### DISCUSSION TITLE
How to adjust seasonality legends and range, seems to be stuck with the original ds range

### DISCUSSION THREAD

**Discussion by TheFern2** [ourownstory/neural_prophet#1098] [General]
Using version 0.5.0 with python 3.10. I am getting unexpected x axis values for seasonality:

![image](https://user-images.githubusercontent.com/10265682/210882060-5a5dbb07-b1f9-4438-8517-64a6d8a99fa5.png)

The tutorial I followed is almost two years old, but I couldn't find any differences from the documentation. I was expecting yearly seasonality to show 1 year and weekly to show 1 week but it shows the entire range of the initial data (ds).

Tutorial result:

![image](https://user-images.githubusercontent.com/10265682/210882372-b0431d0b-4bbf-47a5-ac3a-6a096df4cb78.png)

The code:

![image](https://user-images.githubusercontent.com/10265682/210882776-17c478bd-ad64-438c-a134-8c3acccf3356.png)

```python
m = NeuralProphet()
metrics = m.fit(data, epochs=1000)

# INFO - (NP.df_utils._infer_frequency) - Major frequency D corresponds to 99.694% of the data.
# INFO - (NP.df_utils._infer_frequency) - Dataframe freq automatically defined as D
# INFO - (NP.config.init_data_params) - Setting normalization to global as only one dataframe provided for training.
# INFO - (NP.utils.set_auto_seasonalities) - Disabling daily seasonality. Run NeuralProphet with daily_seasonality=True to # override this.
# INFO - (NP.config.set_auto_batch_epoch) - Auto-set batch_size to 32
# WARNING - (py.warnings._showwarnmsg) - /Users/fernandob/Projects/DataSpell/lib/python3.10/site-packages/pytorch_lightning/trainer/trainer.py:1789: UserWarning: MPS available but not used. Set `accelerator` and `devices` using `Trainer(accelerator='mps', devices=1)`.
 # rank_zero_warn(

# WARNING - (NP.config.set_lr_finder_args) - Learning rate finder: The number of batches (72) is too small than the required number for the learning rate finder (234). The results might not be optimal.

future = m.make_future_dataframe(data, periods=900)
forecast = m.predict(future)
forecast.head()
plot1 = m.plot(forecast)
plot2 = m.plot_components(forecast) # seasonality
```
